Gemfile

```diff
@@ -53,6 +53,7 @@ gem "browser", "~> 5.3", ">= 5.3.1"
 gem "bcrypt", "~> 3.1", ">= 3.1.18"
 gem "maintenance_tasks", "~> 2.1"
 gem "strong_migrations", "~> 1.6"
+gem "zlib", github: "ruby/zlib", ref: "refs/pull/61/head"
```

Comment:
Does pointing to the PR ref mean it will randomly fail at some point when the PR is merged (am I being too optimistic? 🤓)